src/autoyast-rnc/auth-client.rnc

```diff
@@ -1,301 +0,0 @@
-default namespace = "http://www.suse.com/1.0/yast2ns"
```

Comment:
why it this file removed? It can be tricky to do it as it can break autoyast installation.